content/docs/2.7/scalers/new-relic.md

```diff
@@ -68,6 +74,14 @@ spec:
   - parameter: queryKey
     name: new-relic-secret
     key: apiKey
+  secretTargetRef:
```

Comment:
`secretTargetRef` is an array of objects with the 3 keys, duplicating the `secretTargetRef` is incorrect. Could you update this?